src/services/__tests__/OrderService.test.ts

```diff
@@ -0,0 +1,111 @@
+import { Model } from '../../models';
+import { OrderModel, Product } from '../../types';
+import { OrderService } from '../order';
+
+// @ts-ignore for test purposes
+const db = {
```

Comment:
Let's try to use jest.mock() for the db. I assume this allow us to avoid a bunch of `@ts-ignore`